app/modules/submissions/models.py

```diff
@@ -239,6 +239,38 @@ def git_clone(self, project):
 
         return repo
 
+    def import_tus_files(self, transaction_id=None):
+        self.ensure_repository()
+        # @todo duplicates Jons latest tus_upload_dir() to be replaced when available
+        if transaction_id:
```

Comment:
i rewrote `tus_upload_dir()` and moved it a bit if you want to check it out here.  it now handles the 3 flavors it can build on all internally to avoid this if/else stuff.   see it here:
https://github.com/WildMeOrg/houston/blob/sighting_passthrough_basic/app/extensions/tus/__init__.py#L61

Reply:
looking further, i think the only tweak would be the case that when `transaction_id` is None you would want to send the self.guid into this call instead.  maybe something like:
```python
sub_id = None
if transaction_id is None:
    sub_id = self.guid
upload_dir = tus_upload_dir(app, transaction_id=transaction_id, submission_guid=sub_id)

Reply:
We've gone way past this point in discussion so resolving this comment now